src/services/tracking/actions.js

```diff
@@ -1,8 +1,22 @@
-export const LOCATION_CHANGE = 'LOCATION_CHANGE';
+export const INIT_TRACKER = 'INIT_TRACKER';
+export const TRACK_PAGE = 'TRACK_PAGE';
+export const TRACK_EVENT = 'TRACK_EVENT';
 
-export const locationChange = url => ({
-  type: LOCATION_CHANGE,
+export const initTracker = () => ({
+  type: 'INIT_TRACKER',
+});
+
+export const trackPage = pathName => ({
+  type: 'TRACK_PAGE',
+  payload: {
+    pathName,
+  },
+});
+
+export const trackEvent = ({ eventName, trackingParams }) => ({
```

Comment:
```suggestion
export const trackEvent = ({ eventName = 'Click', trackingParams }) => ({
```

근데 이것보단 `EventNames`에 있는 값을 쓰는 게 좋을 것 같기도 한데요 🤔